### PR TITLE
Nothing personal updates (fixes #15420)

### LINKED ISSUE #15420: Update Nothing Personal Landing Page
## Description

We are updating `https://www.mozilla.org/en-US/firefox/nothing-personal/` 

Design: [Figma](https://www.figma.com/design/QyN625zQl1CWshXiow5NcI/N.A.-Fx-Campaign?node-id=6288-86224&t=SPHEvOJJVzY44cS2-1) (look at the 10/24 section)

Copy: [document](https://docs.google.com/document/d/1CHmcOHN3_u-HROr9RMRZ1By8y8TQveaghlt_epy1lr4/edit?usp=sharing)

Contacts: please reach out to Louise Morgan with questions and clarifications.

Notes:
- the behavior of the 'clickity click' section is that it loads one of the 21 options  in the copy document, and then loads, at random, another one of the options on click.
- English only

---

#### Success Criteria
- [ ] any external link should have UTM parameters
- [ ] Please check with Dan Brown and Stephanie Hobson for recent updates to even tagging for campaign pages.

### PR DESCRIPTION
**Pull request by maureenlholland**
## One-line summary

Add new content to the `nothing-personal` page.

- [ ] I used an AI to write some of this code.

## Significant changes and points to review

- New "things" section with button that displays random thing on click
- New "privacy" section
- New "features" section
- Updated fox gif placement
- New link for MyLife trash icon
- [update] no easy switching feature on mobile, as it is not available for mobile browser yet

There should be no change to header styling or functionality.
No change to bottom sticky note functionality.
No missing or broken easter eggs!

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/15420

## Testing

⚠️ ensure switch M24_WEBSITE_REFRESH is OFF

Figma 10/24 [Mozilla only]: https://www.figma.com/design/QyN625zQl1CWshXiow5NcI/N.A.-Fx-Campaign?node-id=6801-8061&node-type=text

design feedback for reference [Mozilla only]: https://docs.google.com/document/d/1-N_Hu7kpQMI46JsuRLTY0Xs-PVQofwXnq2yCN3P_Pyc/edit?tab=t.0 

http://localhost:8000/en-US/firefox/nothing-personal/

- Cross-browser compatibility
- Screen size responsiveness
- No JS experience
   - "Clickity Click" button is gone, there's a details section to expand for more random things
- Reduced motion preference experience
   - when reduced, gifs are static images, bottom sticky is in view, and the only animation is stars fading in on button
   - when allowed, all boxes pop in, bottom sticky slides in, the Clickity Click button stars do a little spinny jump, there's a quick shine over the button and an ongoing pulse

### CODE REVIEW

**Comment by maureenlholland** [comment]
TODO:
- button stateful styling
- add new assets **(need Figma access)**
- check animation behaviour (maybe slide in the fox gif?)
- update gif positioning
- make font sizes and spacing consistent **(need Figma access)**

**Comment by ingramjr** [comment]
* Sticky CTA at top of page
  * Was there a request to change it from 'Download Firefox' to 'Learn more'? 
* Body copy
  * What are your thoughts on converting all the body copy that is currently 400 to 500. 400 weight is coming across a little too light. 
* Browser windows w/ the blue gradient bar
  * Can we make the stroke color #333336
  * Can we see what the stroke weight looks slightly lighter weight? 
  * Make the corner radius' same as the radius' of the 4 feature boxes. 
* Stickers (foxy, heart, smiley face)
  * Make the heart and smiley relative in size to the foxy sticker. 
  * The foxy sticker stroke weight looks different from the heart and smiley. Possibly to make it consistent? 
* Clickity Click Button
  * Hover state - I like the shine! Could it also slightly increase in size? 
  * Hover state - On hover, would it suffice to alter color slightly to signifiy hover? 
  * Active state - Love the depress animation
  * General note - It would be great to add some sort of motion to catch the attention. Would love your thoughts - could it be a light pulsing, the shine animates every 5 seconds, The 3 star shines rotating? All 3!? 
* What would Firefox do browser window
  * Whats your thoughts on making the text box that re-populates every click the 2-line text height instead of having it alternating between the 1-line height and 2-line? Just so that the browser window doesn't resize with every click? 
* Foxy in bush gif
  * I like it on infinite loop. Is it possible to have it pause for 1-2 seconds before looping? Just to give it a beat. 
  * Could we increase the size by 10-15%
  * Make the box strokes same weight as other windows once adjusted. 
* Animations
  * I love the way the windows pop in. Feels natural to the context of it being a desktop! 
  * First window - possible to have the stickers animate in separately/staggered on their own to insinuate them getting stuck on? 
* Newsletter browser window (Mobile Only)
  * Make "Newsletter" centered instead of right justified.

**Comment by slightlyoffbeat** [comment]
@stephaniehobson could you take a look at this through a GA4/events point of view?

File: bedrock/firefox/templates/firefox/nothing-personal/index.html
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/nothing-personal/index.html:214]: It has always bothered me that this is `>>>>` not `&gt;&gt;&gt;&gt;` Can you change them since you're in there 🙏

File: bedrock/firefox/templates/firefox/nothing-personal/index.html
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/nothing-personal/index.html:218]: **Analytics:** Download buttons are good 👍

File: media/js/firefox/nothing-personal/random-answers.es6.js
Comment (stephaniehobson) [inline media/js/firefox/nothing-personal/random-answers.es6.js:23]: **Analytics:** An event for this will be covered by adding `data-cta-text` to the button that triggers it so no action is needed here.

**Review by stephaniehobson — COMMENTED** [review]
Just had a look at analytics stuff 👀

File: media/css/firefox/nothing-personal/_browser.scss
Comment (maureenlholland) [inline media/css/firefox/nothing-personal/_browser.scss:29]: sanity check this tomorrow (still need to remove on mobile in things section, but not sure this method is worth keeping)

File: media/css/firefox/nothing-personal/styles.scss
Comment (maureenlholland) [inline media/css/firefox/nothing-personal/styles.scss:171]: I think this is the Mozilla font-size scale on Firefox themed page, so the custom property vars were slightly off from Figma in ways that seemed worth overriding for this case

**Comment by maureenlholland** [comment]
DO NOT MERGE until rebased to single commit after code review & we have green light from Firefox marketing team

File: media/css/firefox/nothing-personal/styles.scss
Comment (maureenlholland) [inline media/css/firefox/nothing-personal/styles.scss:148]: I thought this would help avoid some absolute positioning, but I've got to push the aside section over the main section padding anyway, so I'm on the fence about using grid over absolute positioning here. Reviewer's choice!

File: bedrock/firefox/templates/firefox/nothing-personal/index.html
Comment (craigcook) [inline bedrock/firefox/templates/firefox/nothing-personal/index.html:55]: Suggestion: This `span` could be an `em` for semantic emphasis as well as presentation (and make it all-caps and non-italic in CSS)

File: bedrock/firefox/templates/firefox/nothing-personal/index.html
Comment (craigcook) [inline bedrock/firefox/templates/firefox/nothing-personal/index.html:59]: Is this still TODO? Or can this comment be removed?

File: bedrock/firefox/templates/firefox/nothing-personal/index.html
Comment (craigcook) [inline bedrock/firefox/templates/firefox/nothing-personal/index.html:273]: Nit: can remove some of the additional tracking params from this URL:
```suggestion
    <a class="c-aside-link" href="https://www.youtube.com/watch?v=F-nFQryDB0s" aria-label="Introducing the new Firefox...jk. It's red pandas" target="_blank" rel="external noopener" data-link-text="MyLife">
```

File: bedrock/firefox/templates/firefox/nothing-personal/index.html
Comment (craigcook) [inline bedrock/firefox/templates/firefox/nothing-personal/index.html:103]: This is a really nice fallback for non-JS. 👏

File: media/css/firefox/nothing-personal/_things.scss
Comment (craigcook) [inline media/css/firefox/nothing-personal/_things.scss:199]: Should this button also get `cursor: pointer;` to help hint at clickability? (Arguably the finger should only be for links but we do apply it on other non-link interactive buttons like form submits, so maybe it's universally understood as "I can click this"?)

**Review by craigcook — APPROVED** [review]
This all looks awesome, really nice work. Just a few minor nits, r+wc 🦊

File: media/css/firefox/nothing-personal/_things.scss
Comment (maureenlholland) [reply media/css/firefox/nothing-personal/_things.scss:199]: I like the pointer, does seem to help indicate clickability

**Pull request merged**
